kolibri/plugins/perseus_viewer/buildPerseus.js

```diff
@@ -0,0 +1,119 @@
+const fs = require('fs');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const mkdirp = require('mkdirp');
+const rimraf = require('rimraf');
+const webpack = require('webpack');
+const buildConfig = require('./perseusBuildConfig');
+const extractPerseusMessages = require('./extractPerseusMessages');
+
+const currentCwd = process.cwd();
+
+const submodulesPath = path.resolve(__dirname, './submodules');
+
+rimraf.sync(submodulesPath);
+
+mkdirp.sync(submodulesPath);
+
+process.chdir(submodulesPath);
+
+console.log('Cloning the Learning Equality Perseus repo');
+
+execSync('git clone https://github.com/learningequality/perseus.git');
+
+console.log('Learning Equality Perseus repo successfully cloned');
+
+process.chdir('perseus');
+
+console.log('Fetching all perseus submodules');
+
+execSync('git submodule update --init --recursive');
+
+console.log('All perseus submodules fetched');
+
+console.log('Installing perseus dependencies');
+
+execSync('yarn install');
+
+console.log('Perseus dependencies installed');
+
+process.chdir(__dirname);
+
+const target = path.resolve(__dirname, './assets/dist');
+
+console.log('Clearing dist folder');
+
+rimraf.sync(target);
+
+mkdirp.sync(target);
```

Comment:
Trying to make this a bit easier to read

```suggestion
console.log('Cloning the Learning Equality Perseus repo...');
const submodulesPath = path.resolve(__dirname, './submodules');
rimraf.sync(submodulesPath);
mkdirp.sync(submodulesPath);
process.chdir(submodulesPath);
execSync('git clone https://github.com/learningequality/perseus.git');

console.log('Fetching all perseus submodules...');
process.chdir('perseus');
execSync('git submodule update --init --recursive');

console.log('Installing perseus dependencies...');
execSync('yarn install');

console.log('Clearing dist folder...');
process.chdir(__dirname);
const target = path.resolve(__dirname, './assets/dist');
rimraf.sync(target);
mkdirp.sync(target);
```